### ISSUE TITLE
Mechanism to disable or neutralize vote buying for community note users

### ISSUE THREAD

**Issue by Licsak25** [twitter/communitynotes#164]
There is an account with a large number of followers that asked people to vote "useless" on the community notes attached to their posts.  

https://twitter.com/tezheya/status/1724378192977498289

In fact, notes that were supposed to be "useful'' at one point were rendered "useless'' due to organized voting actions.

Can Community Notes withstand these vote-buying practices?

**Comment by elvey** [comment]
Looking into this, brings up another issue which is that there’s no facility for translating community notes, unlike tweets. Yeah, I could cut and paste into a translation app or use a browser that can translate the whole page but no reason they should be harder to translate than tweets — with a click.

**Comment by Intina47** [comment]
What if we added a layer of accountability to Community Notes? Imagine if, when someone dislikes a note, they have to provide a detailed reason. These reasons would be publicly reviewed, and if a dislike reason is not approved, a full report explaining the decision would be available for users to check.
To avoid bias, those creating reports would be validated and held accountable for their feedback. Dislikes would need a certain number of approvals to be considered valid, and a summary report of why it's approved would be provided.

**Comment by armchairancap** [comment]
> Can Community Notes withstand these vote-buying practices?

That's "vote brigading", not "vote buying".

Mere vote brigading already doesn't work as a note must be supported (or not) by voters with diverse backgrounds. If Community Notes can't ensure diversity in votes then this approach can't work, but that problem should be filed a different issue.

**Comment by BDPershing** [comment]
*shock* *pulse noises*
Both vote brigading and vote buying is prevalent way to influence the CN system.
First question is how does CN even determine "diverse backgrounds?"
Is it based on those being followed? The voter? What determines the diversity of the voter? Is it based on their stated description?

First way to avoid such an issue is if the voter is subscribed/following the person who has a CN pending on them. Those following the person with the CN on their post will be more willing to downgrade/vote negative on that note. Is it a guarantee? No, since people hate follow aswell. Though if they're hate following it could lead to a useful way of preventing CN abuse by individuals who just don't like the targeted user.
So my suggestion is a way to weigh those who are subscribed/following the individual with a CN on their posts. Less than those who are not, chances of those not subscribed to the content will be more open to the CN rather than using the opportunity to drown CN that negatively impact the poster.

**Comment by armchairancap** [comment]
> First question is how does CN even determine "diverse backgrounds?"

Well, it does that already, doesn't it? IIRC it compares each voters' voting history. 
If a CN is supported by a bunch of people with similar voting record and no one with a different voting history, I doubt that CN would get published. 

> What determines the diversity of the voter? 

I don't know, but in any case that's not the topic here. "Diversity of opinions based on voting record" is my personal view. 
Someone who doesn't vote in a partisan way (i.e. they have a record of "bipartisanship" or "multipartisanship") provides diversity more than two voters with a history of consistent partisanship.

> my suggestion is a way to weigh those who are subscribed/following the individual with a CN on their posts.

That's fine and could improve overall results. 
It wouldn't do much for vote buying (which I still think is a wrong term, because even if you pay, how do you even know if the someone voted Not Helpful?) because if anyone did it they wouldn't be stupid enough to get caught and they'd automate it or outsource to an automated service. Manually sourcing and paying hundreds of people, each of whom can report you to X, is a stupid idea.

The OP:

> There is an account with a large number of followers that asked people to vote "useless" on the community notes attached to their posts.

Poster A could ask friend B to ask his followers to vote Not Helpful and vice versa. You'd need 3-4 friends who target different niches (gardening, knitting) of the same interest group (grannies), to minimize overlap and you're safe...

I don't think this is a big problem, but I can view proposed CNs as well so if there are opposing views I'd check a few. If a note does NOT get published, that may be a problem for those who can't see proposed CNs. But even in that case, people can post comments or comment in a new post. 

I support the idea to weigh lower the comments of the followers of the account where a CN is proposed, but I wouldn't consider that a preventive measure for vote brigading (or buying) and I suspect something like that is already in place.
